src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionsCache.cs

```diff
@@ -1,126 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.UnifiedSuggestions;
-using Roslyn.Utilities;
-using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
-
-namespace Microsoft.CodeAnalysis.LanguageServer.Handler.CodeActions
-{
-    /// <summary>
-    /// Caches suggested action sets between calls to <see cref="CodeActionsHandler"/> and
-    /// <see cref="CodeActionResolveHandler"/>.
-    /// </summary>
-    internal class CodeActionsCache : ILspService
```

Comment:
happy to see thsi go.  but what made that possible?

Reply:
discussed offline - the cache was never read from (it recomputes instead)